src/main/java/seedu/address/model/activity/Activity.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.model.activity;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Represents an Activity class containing participants and expenses.
+ */
+public class Activity {
+
+    private final String title;
+    private final ArrayList<Person> participants = new ArrayList<>();
+    private final ArrayList<Expense> expenses = new ArrayList<>();
+
+    /**
+     * Constructor for Activity.
+     * @param title A String representation of the activity name.
+     * @param people The people participating in the activity.
+     */
+    public Activity(String title, Person ... people) {
+        requireAllNonNull(title);
+        this.title = title;
+        for (int i = 0; i < people.length; i++) {
+            participants.add(people[i]);
```

Comment:
We can consider using the static method `addAll` in Java's Collection class when adding multiple elements at once: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Collections.html#addAll(java.util.Collection,T...)

Reply:
`people` is an array so the loop avoids a little bit of overhead. `addAll` requires conversion to `Collection`.